tar/bsdtar.c

```diff
@@ -1779,6 +1782,11 @@ dooption(struct bsdtar *bsdtar, const char * conf_opt,
 
 		bsdtar->option_quiet = 1;
 		bsdtar->option_quiet_set = 1;
+	} else if (strcmp(conf_opt, "resume-extract") == 0) {
```

Comment:
This code here is for processing configuration file options.  Why do we need it?